content/concepts/transport.md

```diff
@@ -115,3 +115,189 @@ and some places in the codebase still use the "swarm" terminology.
+certificate. They then sign an intermediate chain using their self-signed CA and put it 
+as a certificate chain in the TLS handshake.
+
+At the end of the handshake, each peer knows the certificate of the other. The peer can 
+verify if the connection was established with the correct peer by looking up the first 
+CA certificate on the certificate chain, retreive the public key, and using it to calculate 
+the opposing peer ID. QUIC acts like a record layer with TLS 1.3  as the backend as TLS is 
+responsible for all the cryptography.
+
+{{% notice "info" %}}
+
+To be clear, there is no additional security handshake and stream muxer need as QUIC provides 
+all of this by default.
+
+{{% /notice %}}
+
+Following the multiaddress format described earlier, a standard QUIC connection will
+look like:
+
+```
+/ip4/127.0.0.1/udp/65432/quic/
+```
+
+In this section, we offered an overview of QUIC and how QUIC works in libp2p.
+
+{{% notice "tip" %}}
+
+For more details on QUIC, including its limitations 
+check out the following resources:
+
+{{% /notice %}}
+
+## WebTransport
+
+Another transport protocol under development at the IETF is WebTransport.
+WebTransport is a new specification that uses QUIC to offer an alternative to
+WebSockets. Instead, it can be considered WebSockets over QUIC by allowing 
+browsers to establish a stream-multiplexed and bidirectional 
+connection to servers. 
+
+The specification can depend on and reuse the QUIC infrastructure in place 
+to offer WebSockets that take the benefits of UDP and offer sockets without head-of-line 
+blocking.
+
+Recall that WebSockets are bidirectional, full-duplex communication between two 
+points over a single-socket connection. WebTransports, as a result, can be used
+like WebSockets, but with the support of multistreams.
+
+WebTransport streams can be arbitrary in size and independent when possible. 
+They are reliable but can be canceled when possible. The datagrams in a 
+WebTransport connections are MTU-sized and can be unreliable when possible.
+
+{{% notice "caution" %}}
+
+There is a functioning WebTransport implementation in go-libp2p that is part 
+of the v0.23 release.
+
+The implementation should be used experimentally and is not recommended for
+production environments.
+
+{{% /notice %}}
+
+For network stacks like libp2p, WebTransport is a pluggable
+protocol that fits well with a modular network design.
+
+For a standard WebSocket connection:
+
+- 1 RTT for DNS resolution
+- 1 RTT for TCP handshake
+- 1 RTT for TLS handshake
+- 1 RTT for WebSocket handshake
+- 1 RTT for Multistream Security handshake
+- 1 RTT for libp2p handshake
+
+Plenty of handshakes and roundtrips: 6 RTTs: 5 handshakes + 1 DNS resolution
+
+WebTransport running over QUIC only requires 4 RTTs, as:
+
+- 1 RTT for QUIC handshake
+- 1 RTT for WebTransport handshake
+- 2 RTT for libp2p handshake; one for multistream and one for the secure 
```

Comment:
QUIC provides the security, we don't need to add TLS or noise here (we don't double encrypt).

We _do_ need to authenticate the connection and do that with a Noise handshake (see webtransport spec)

Reply:
Addressed